rest_framework_gis/fields.py

```diff
@@ -67,6 +62,18 @@ class GeoJsonDict(OrderedDict):
     Used for serializing GIS values to GeoJSON values
     TODO: remove this when support for python 2.6/2.7 will be dropped
     """
+    def __init__(self, *args, **kwargs):
+        """
+        If a string is passed attempt to pass it through json.loads, because it should be a geojson formatted string.
```

Comment:
Could you please break the comment like this?
```
If a string is passed attempt to pass it through json.loads,
because it should be a geojson formatted string.
```